clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/task/TaskAware.groovy

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 Target, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.openstack.task
+
+import com.netflix.spinnaker.clouddriver.data.task.Task
+import com.netflix.spinnaker.clouddriver.data.task.TaskRepository
+
+trait TaskAware {
+  static final String UPSERT_LOADBALANCER_PHASE = "UPSERT_LOAD_BALANCER"
```

Comment:
This task is a way to add TaskStatus behavior to concrete classes ... The PHASE definition it needed by the taskStatus to provide a context, therefore, I defined here so that it could be easily accessible.